### PR TITLE
Fix incsearch ctrl-g/t

### PR DESCRIPTION
**Pull request by mattn**
Insert word on cursor does not work after CTRL-G/T while incsearch.

```
aAA, 112BBB222, CCC333, DDD, EEE
aaa, 111bbb222, ccc333, ddd, eee
```

`set incsearch` and type `/ddd<C-G><C-R><C-W>`. My expected result is `/dddeee` but insert word does not work since cursor location is not at end of match.

screenshot1

 This fix move the corsor-want to the end of match.

### CODE REVIEW

File: src/os_win32.c [hunk 1]
```diff
@@ -3713,7 +3713,12 @@ ResizeWindow(
     HANDLE     hConsole,
     SMALL_RECT srWindowRect)
 {
-    if (!SetConsoleWindowInfo(hConsole, TRUE, &srWindowRect))
+    {
+	FILE* fp = fopen("testtest.log", "a");
+	fprintf(fp, "%d,%d,%d,%d\n", srWindowRect.Left, srWindowRect.Right, srWindowRect.Top, srWindowRect.Bottom);
```
Comment (mattn) [inline src/os_win32.c]: Sorry. I'll remove this.

**Comment by mattn** [comment]
Is  this test expected?

```
func Test_search_cmdline5()
  if !exists('+incsearch')
    return
  endif
  " Do not call test_override("char_avail", 1) so that <C-g> and <C-t> work
  " regardless char_avail.
  new
  call setline(1, ['  1 the first', '  2 the second', '  3 the third'])
  set incsearch
  1
  call feedkeys("/the\<c-g>\<c-g>\<cr>", 'tx')
  call assert_equal('  3 the third', getline('.'))
  $
  call feedkeys("?the\<c-t>\<c-t>\<c-t>\<cr>", 'tx')
  call assert_equal('  2 the second', getline('.'))
  " clean up
  set noincsearch
  bw!
endfunc
```

**Comment by brammool** [comment]
Strange that the test passes on Travis but fails on Appveyor.

**Comment by mattn** [comment]
I think the test was wrong since `$` goes first letter of third line. So typing `<c-t>` twice should goes `2 the second`.

**Comment by mattn** [comment]
Probably, wrapscan is enabled on appveyor.

**Pull request closed without merging**